scene_objects:
  - id: environment
    mesh: "resource/environment/cage_closed_1m.stl"
    frame: "world"
    position: [0.0, 0.0, 0.0]     # nur Floats
    rpy_deg: [0.0, 0.0, 0.0]

  - id: substrate_mount
    mesh: "resource/substrate_mounts/substrate_mount_flat_240x240x50.stl"
    frame: "world"
    position: [0.0, 0.0, 0.0]
    rpy_deg: [0.0, 0.0, 0.0]

  - id: substrate
    mesh: "resource/substrates/wafer_d200_h2.stl"
    frame: "scene"     # ganz wichtig
    position: [0.0, 0.0, 0.0]
    rpy_deg: [0.0, 0.0, 0.0]
    optional: true

